src/new-client/src/Web/TearOutSection/handleTearOutSection.web.ts

```diff
@@ -0,0 +1,29 @@
+import { constructUrl } from "@/utils/url"
```

Comment:
This can go in `handleTearOutSection.ts` as we don't need any other platform implementations at the moment.

Reply:
I'm not sure I follow, `handleTearOutSection.web.ts` uses `constructUrl` 

`
export function handleTearOut(symbol: string) {
  openWindow(
    {
      url: constructUrl(`/spot/${symbol}`),
`

If I move this to the generic handleTearOutSection.ts wouldn't I get a warning?

Reply:
What I mean to say is - we don't have any other implementations for handleTearOutSection so we don't need to have a `.web.ts` version, this logic can be moved to `handleTearOutSection.ts`